common/djangoapps/course_modes/models.py

```diff
@@ -166,7 +166,7 @@ def paid_modes_for_course(cls, course_id):
         return [mode.to_tuple() for mode in found_course_modes]
 
     @classmethod
-    def modes_for_course(cls, course_id, only_selectable=True):
+    def modes_for_course(cls, course_id, include_expired, only_selectable=True):
```

Comment:
Please make this a keyword argument with default `False` and update the docstring.